Documentation/markdown/SupportedIFCrepresentations.md

```diff
@@ -24,7 +24,92 @@ Meaning of columns:
+| `IfcAxis2Placement2D`                 | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcAxis2Placement3D`                 | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcAxis2PlacementLinear`             | :x:                  | `PlacementConverter`       | -                     | -     |
+| `IfcBlock`                            | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcBlossCurve`                       | :x:                  | `CurveConverter`           | -                     | -     |
+| `IfcBooleanClippingResult`            | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcBooleanResult`                    | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcBoundedCurve`                     | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcBoundingBox`                      | :heavy_check_mark:   | `SolidModelConverter`      | -                     | -     |
+| `IfcBoxedHalfSpace`                   | :clock9:             | `SolidModelConverter`      | -                     | -     |
+| `IfcBSplineCurveWithKnots`            | :heavy_check_mark:   | `SplineConverter.`         | -                     | -     |
+| `IfcCartesianPoint`                   | :heavy_check_mark:   | `PlacementConverter`       | :heavy_check_mark:    | -     |
+| `IfcCartesianPointList2D`             | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCartesianPointList3D`             | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCartesianTransformationOperator2D`| :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCartesianTransformationOperator3D`| :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCartesianTransformationOperator2DnonUniform` | :heavy_check_mark:   | `PlacementConverter`       | -                     | -     |
+| `IfcCircle`                           | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCircularArcSegment2D`             | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcClothoid`                         | :x:                  | `CurveConverter`           | -                     | [#304](https://github.com/tumcms/Open-Infra-Platform/issues/304) |
+| `IfcCompositeCurve`                   | :heavy_check_mark:   | `CurveConverter`           | -                     | -     |
+| `IfcCompositeCurveOnSurface`          | :x:                  | -                          | -                     | -     |
+| `IfcCompositeCurveSegment`            | :x:                  | `CurveConverter`           | -                     | -     |
```

Comment:
I believe this one's supported. Double-check, please.

Reply:
IfcCompositeCurveSegment isn't supported (see CurveConverter->convertIfcSegment)

Reply:
Right, I see the confusion.

It is supported for 4x1 and 4x3_rc1, but not (yet) for 4x3_rc3. Mark as supported.

Reply:
Resolved 31a5bd4